django_fido/admin/authenticator.py

```diff
@@ -0,0 +1,108 @@
+"""Admin for django_fido authenticator."""
+from __future__ import unicode_literals
+
+from django import forms
+from django.conf.urls import url
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.http import Http404, HttpResponse, HttpResponseRedirect
+from django.urls import reverse, reverse_lazy
+from django.utils.translation import ugettext_lazy as _
+
+from django_fido.forms import Fido2RegistrationForm
+from django_fido.models import Authenticator
+from django_fido.views import Fido2RegistrationRequestView, Fido2RegistrationView
+
+
+class Fido2RegistrationAdminForm(Fido2RegistrationForm):
+    """Registration form with user selection."""
+
+    user = forms.ModelChoiceField(
+        queryset=get_user_model().objects.all(),
+        error_messages={'required': _("Operation wasn't completed.")},
```

Comment:
The message doesn't seem to be matching the error code.

Reply:
However, it is the same code and message that all other fields of the same form use. Maybe they deserve to be replaced with something more informative, but I don't believe it should be part of this PR.

Reply:
Looks like a confusion.

Reply:
We discussed it and I deleted the error message altogether. The default one will be used.